### PR TITLE
Create and add curator and koku applications to nerc-ocp-test cluster 

### PR DESCRIPTION
**Pull request by dystewart**

### CODE REVIEW

File: clusters/nerc-ocp-test/koku-metrics-operator/application.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/nerc-ocp-config.git
+    targetRevision: HEAD
+    path: koku-metrics-operator/overlays/nerc-ocp-test
+  destination:
+    name: nerc-ocp-test
+    namespace: koku-metrics-operator
+  syncPolicy:
```
Comment (larsks) [inline clusters/nerc-ocp-test/koku-metrics-operator/application.yaml:14]: Unless there's a specific reason this needs to be different, **do not** set the `syncPolicy` here in the application; instead, add the `nerc.mghpcc.org/sync-policy: common` label to the resource so the policy is added by the parent `kustomization.yaml`

**Review by larsks — CHANGES_REQUESTED** [review]
Arg, pressed the wrong button so failed to mark this as "request changes"; sorry for the noise...

File: clusters/nerc-ocp-test/curator/application.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/nerc-ocp-config.git
+    targetRevision: HEAD
+    path: curator/overlays/nerc-ocp-test
+  destination:
+    name: nerc-ocp-test
+    namespace: curator
+  syncPolicy:
```
[the remark this answers is not in the document]
Comment (dystewart) [reply clusters/nerc-ocp-test/curator/application.yaml]: Ah, forgot about that patch, updated

File: clusters/nerc-ocp-test/koku-metrics-operator/application.yaml (hunk 1 above)
Comment (dystewart) [reply clusters/nerc-ocp-test/koku-metrics-operator/application.yaml]: Ah, forgot about that patch, updated

**Review by larsks — APPROVED** [review]

**Pull request merged**